pkg/constants/http_test.go

```diff
@@ -24,6 +24,20 @@ func TestURLConstruction(t *testing.T) {
 			},
 			expectedURL: "/data/db1/key1",
 		},
+		{
+			name: "ExecuteQuery",
+			execute: func() string {
+				return URLForDataQuery("db1")
+			},
+			expectedURL: "/data/db1",
+		},
```

Comment:
why do we need this? I don't see this path ever being used at the moment

Reply:
Done